presto-docs/src/main/sphinx/release/release-0.218.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.218
+=============
+
+General Changes
+---------------
+
+* Add support for using binary (smile) encoding for coordinator-to-worker communication.
+This feature is experimental, and it can be enabled with the ``experimental.internal-communication.binary-transport-enabled`` config option.
+Enabling this feature should help with coordinator scalability, and should reduce network, cpu, and memory usage on the coordinator.
+* Add ST_Area for SphericalGeography
+* Add a new system table ``system.metadata.analyze_properties`` to display available
```

Comment:
```
* Add a new system table ``system.metadata.analyze_properties`` that stores the
  properties that can be used with the ``ANALYZE`` statement.
```

Reply:
Additions go to the bottom of the release note items per https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines.